eng/imports/Packages.targets

```diff
@@ -19,7 +19,7 @@
     <!-- VS SDK -->
     <PackageReference Update="EnvDTE"                                                                 Version="17.0.32112.339" />
     <PackageReference Update="Microsoft.Internal.VisualStudio.Interop"                                Version="17.1.0-preview-2-32002-062" />
-    <PackageReference Update="Microsoft.VSSDK.BuildTools"                                             Version="17.1.1035-preview2" />
+    <PackageReference Update="Microsoft.VSSDK.BuildTools"                                             Version="17.3.2093" />
```

Comment:
This was required to be updated as the previous version was causing build failures after a maintenance reset/upgrade cycle that just happened.